src/index.html

```diff
@@ -2,9 +2,25 @@
 <html>
 
 <head>
-    <title>
-        My first web page
-    </title>
+
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Learn about job openings at starlight and submit your application now">
+    <meta name="keywords" content="Cyber security, software development, jobs">
+    <meta name="author" content="Starlight">
+    <meta name="robots" content="index, follow">
+    <meta property="og:title" content="Star light">
+    <meta property="og:description" content="Learn about job openings at starlight and submit your application now">
+    <meta property="og:image" content="image2.jpeg">
+    <meta name="twitter:title" content="Starlight">
+    <meta name="twitter:description" content="Learn about job openings at Jovian and submit your application now">
+    <meta name="twitter:image" content="image2.jpeg">
+    
+
+
+    <title>Star light</title>

```

Comment:
Make the title a little longer

Reply:
how many character do you suggest for the title ?

Reply:
at least 12